Dshop/apps/products_catalogue/templates/products_catalogue/product_detail.html

```diff
@@ -3,9 +3,13 @@
 {% load thumbnail %}
 
 {% block content %}
-<!--Lightbox2 https://lokeshdhakar.com/ Lightbox2 is licensed under The MIT License-->
-<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/lightbox2/2.11.3/css/lightbox.min.css">
 <script src="https://cdnjs.cloudflare.com/ajax/libs/lightbox2/2.11.4/js/lightbox-plus-jquery.js"></script>
+<script>
+    lightbox.option({
```

Comment:
This is one place, but also the image in the template has to be scaled down using this solr thumbnail plugin. 
Ie: here: 
`<a href="{{ pic.image.url }}" data-lightbox="image-1">` it should use scaled image, not original one. You are right, it won't save any bandwidth because the image is still huge.